### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by danzee1** [projectdiscovery/nuclei#2060] [General]
Description:

Many times due to network error or VPS errors Nuclei scan can stop ... It will be great if scanner show that which or how many of the total items/hosts were scanned ....  So that we can remove those already scanned ips/hosts from txt file ... 

Thanks !!

**Comment by ehsandeep** [comment]
@attacker007, there is `-resume` feature that already supports the same when you stop the scan using CLT + C, unfortunately, nuclei can not know if the scan is stopped for any unknown reasons or network issue on your system, instead, I will suggest using `screen` session to run your scan to avoid these accidental stops.
